### ISSUE TITLE
ReadMe still references elm-package.json and copying dependencies

### ISSUE THREAD

**Issue by Chadtech** [elm-explorations/test#19]
Hi,

The readme still talks about elm-package.json, even thought 0.19 uses elm.json files. And it still talks about copying dependencies between your main and test elm-package.json files, even though we can use one elm.json in 0.19

I would make the PR myself, but I am currently trying to figure out how to do tests with 0.19 myself, so I dont really know how to instruct others.

**Comment by rtfeldman** [comment]
Good point, thanks! It's on the list for https://github.com/rtfeldman/node-test-runner/pull/265 to change the behavior of `elm-test init` to no longer attempt to copy dependencies, but 👍 to the need for a doc fix.

**Comment by agmenc** [comment]
@Chadtech Did you manage to figure out how to run tests in 0.19? I'm still stumped.

Whenever I run `elm-test tests/Example.elm` from my project's root directory, I get:
> It looks like you're running elm-test from within your tests directory.
> Please run elm-test from your project's root directory, where its elm-package.json lives!

I've followed the suggestions in [this discourse post](https://discourse.elm-lang.org/t/elm-test-for-0-19/1748/11) and taken the following steps:
- Used [elm-rosetree](https://github.com/zwilias/elm-rosetree) as a reference to copy from, because the author has upgraded to Elm 0.19, has tests and has a passing build.
- Installed the latest elm-test beta `npm install elm-test@0.19.0-beta4`
- Cleared existing builds `rm -r elm-stuff`
- Run `elm-test init`. This doesn't really seem to be working well with Elm 0.19, because it is trying to use the `elm-package` command, which doesn't exist in 0.19.
- Removed the extra `elm-package.json` installed in `tests` by `elm-test init`. The updated projects I used as references don't have `elm-package.json` or `elm.json` in their `tests` directory, so I figure it is no longer needed. Having it there made no difference, anyway.
- I've tried creating a spurious (but correct) `elm-package.json` in my root directory, but I see the same error message.

I'd be grateful for any suggestions.
